packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -0,0 +1,70 @@
+import { inject, observer } from 'mobx-react'
+import { toStream } from 'mobx-utils'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { from } from 'rxjs'
+
+import InteractionLayer from './InteractionLayer'
+
+function storeMapper (stores) {
+  const { drawing } = stores.classifierStore
+  return {
+    drawing
+  }
+}
+
+@inject(storeMapper)
+@observer
+class InteractionLayerContainer extends Component {
+  constructor () {
+    super()
+    this.onMouseDown = this.onMouseDown.bind(this)
+    this.onMouseMove = this.onMouseMove.bind(this)
+    this.onMouseUp = this.onMouseUp.bind(this)
+  }
+
+
+  componentDidMount () {
+    const stream = from(toStream(() => this.props.drawing.mouseEventStream))
+    stream.subscribe(z => console.log(z))
```

Comment:
Maybe leave an inline to do comment if there's going to be a defined callback here later?

Reply:
I've added one, but it's not showing up in this comment diff - you can see it at https://github.com/zooniverse/front-end-monorepo/pull/170/files#diff-b5aee330efdbec039e4275348c0a1bc0R27